pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -101,6 +101,12 @@ var numRestoreWorkers = settings.RegisterIntSetting(
 	settings.PositiveInt,
 )
 
+var restoreAtNow = settings.RegisterBoolSetting(
```

Comment:
Doesn't look like we replace the restored timestamps with `WriteAtRequestTime` in the `restoreDataProcessor`.

Should we assert the use of this timestamp in `UpdateSSTTimestamps`? I don't necessarily think we _need_ to use `WriteAtRequestTime` at all -- I'd be fine with just replacing whatever timestamps we're given -- but if we add a special timestamp like this then let's require it.

Reply:
> Doesn't look like we replace the restored timestamps with WriteAtRequestTime in the restoreDataProcessor.

Yeah, it didn't seem like we needed to, since we just batch them as-is into SSTs. `BulkAdder` on the other takes a writeTimestamp with with to construct its own keys, so I had to pass it _something_, thus adding a constant to the library that callers can pass it.

I'm apathetic towards, or a bit disinclined to, require any specific existing timestamp in `UpdateSSTTimestamps` / `AddSSTable`; as it stands, the KV API method takes its direction from the boolean request parameter if it is true, it just doesn't care what timestamps are there, and that seems fine?

To the extent that we have a "special" timestamp here, I was thinking it is only special between the BulkAdder and BulkAdder's users, for the have-to-pass-something reason above.

Reply:
Sure, that seems fine.

Reply:
Removed the special timestamp entirely in favor of a bool param.